h2o-persist-hdfs/src/test/java/water/persist/PersistHdfsTest.java

```diff
@@ -28,11 +32,17 @@
   @BeforeClass
   public static void setup() { stall_till_cloudsize(1); }
```

Comment:
Commercial time / hint: Perfect opportunity to take advantage of the new and improved H2ORunner :)